pack/eoe/eoep.json

```diff
@@ -1029,6 +1095,28 @@
         "type_code": "asset",
         "xp": 5
     },
+    {
+        "code": "08099",
+        "cost": 2,
+        "deck_limit": 2,
+        "faction_code": "seeker",
+        "faction2_code": "rogue",
+        "health": 2,
+        "illustrator": "Vlad Ricean",
```

Comment:
I don't believe we know for sure who the illustrator is right? Best I can find for this card is https://i.imgur.com/4vK8jyl_d.webp?maxwidth=760&fidelity=grand (and the unsanctioned spoiler has a camera flash hiding the illustrator name hahah)